thunder/tests/resnet.py

```diff
@@ -0,0 +1,313 @@
+"""Modified from https://github.com/pytorch/vision/blob/main/torchvision/models/resnet.py"""
+
+from functools import partial
+from typing import Any, List, Optional, Type, Union
```

Comment:
didn't quite check the use but some types are not commonly used in this repo
```suggestion
from typing import Any, Type
```